examples/3_NeuralNetworks/autoencoder.py

```diff
@@ -83,11 +86,51 @@ def decoder(x):
 optimizer = tf.train.RMSPropOptimizer(learning_rate).minimize(cost)
 
 # Initializing the variables
-init = tf.initialize_all_variables()
+init = tf.global_variables_initializer()
```

Comment:
Thank you! It is a good point, my bad, I will close this pull request and submit a new one.